### PR TITLE
Feature/572 recurring reservations failed reservations

### PR DESCRIPTION
**Pull request by lraivio**
This PR:
- Adds failed reservations to state
- Shows failed reservations on reservation success modal below the successful reservations
- Shows fail reasons below every failed reservation

Built on top of #587 so that should be merged first.

Refs #572.

### CODE REVIEW

File: app/state/reducers/ui/reservationsReducer.js [hunk 1]
```diff
@@ -29,6 +29,15 @@ function selectReservationToEdit(state, action) {
   });
 }
 
+function parseError(error) {
+  if (error.response && error.response.non_field_errors && error.response.non_field_errors.length) {
+    return error.response.non_field_errors.join('. ').replace('[\'', '').replace('\']', '');
+  } else if (error.response && error.response.detail) {
+    return error.response.detail;
+  }
+  return 'Jotain meni vikaan';
```
Comment (YoYuUm) [inline app/state/reducers/ui/reservationsReducer.js]: This message is not translated but I guess it is fine, as the backend messages are in finnish too?

**Comment by YoYuUm** [comment]
I've approved this PR accidentally, but it still needs a rebase. After it is done it is good to go.

File: app/state/reducers/ui/reservationsReducer.js (hunk 1 above)
Comment (lraivio) [inline app/state/reducers/ui/reservationsReducer.js]: Yeah, I think it's okay to have that just in Finnish for the moment. The API error messages are also in Finnish and these are only shown to admins. If we at some point make recurring reservations available to regular users as well, then we might translate these as well.

**Comment by lraivio** [comment]
Alright, thanks @YoYuUm. Rebase is now done.

**Pull request merged**